src/components/Library/LibraryBook/LibraryBook.tsx

```diff
@@ -65,7 +65,7 @@ const LibraryBook: React.FC<BookProps> = ({ book, update, onDelete }) => {
           <DoneEl
             _id={book._id}
             rating={book.rating}
-            resume={book.resume}
+            resume={book.resume? book.resume : ''}
```

Comment:
Please, cancel it. On the backend is only string type/ I will change it in Trello.

Reply:
Fixed